test/unit/fixtures/CorporateCampaignContentfulPageRaw.json

```diff
@@ -62,6 +62,7 @@
 			},
 			"fields": {
 				"name": "Promo Campaign Test",
+				"pageBackgroundColor": "",
```

Comment:
This won't come back at all from contentful if the field is undefined in contentful.